lib/pushmi_pullyu/fedora_config.rb

```diff
@@ -0,0 +1,46 @@
+require 'yaml'
+class PushmiPullyu::FedoraConfig
```

Comment:
Are you going to create a class for every config file too? Assuming a SolrConfig class is coming too? Might get a bit much no?